### PR TITLE
Close #15660: Allow showing window buttons on the left

### PR DESCRIPTION
**Pull request by Gymnasiast**
On the right:
afbeelding

On the left:
afbeelding

Resolves #15660.

### CODE REVIEW

File: src/openrct2/config/Config.cpp [hunk 1]
```diff
@@ -38,6 +38,12 @@
 using namespace OpenRCT2;
 using namespace OpenRCT2::Ui;
 
+#ifdef __APPLE__
```
Comment (duncanspumpkin) [inline src/openrct2/config/Config.cpp:41]: Not convinced people will like the button switching to the other side after 22 years of it being on the right.

Comment (AaronVanGeffen) [reply src/openrct2/config/Config.cpp:41]: It's just the default that switches, and only for macOS users. I personally think it's a nice touch, and consistent with OS behaviour.

**Comment by AaronVanGeffen** [comment]
Could you add example screenshots of the buttons on the left-hand and right-hand side?

File: src/openrct2/config/Config.cpp (hunk 1 above)
Comment (ZehMatt) [reply src/openrct2/config/Config.cpp:41]: I think we can default to left side for apple users and see what the reaction is like, I also have a feeling that they gonna appreciate this.

**Comment by Gymnasiast** [comment]
@AaronVanGeffen Sure, added some to the opening post.

**Comment by ZehMatt** [comment]
I believe this needs a changelog entry.

**Comment by AaronVanGeffen** [comment]
> I believe this needs a changelog entry.

Looks like one has been added already

**Comment by ZehMatt** [comment]
> > I believe this needs a changelog entry.
> 
> Looks like one has been added already

Yeah its a single commit so I missed that

**Review by ZehMatt — APPROVED** [review]
Tested it and looks all good

File: src/openrct2/config/Config.cpp (hunk 1 above)
Comment (IntelOrca) [reply src/openrct2/config/Config.cpp:41]: I think OpenTTD always defaults to the left.

**Comment by Broxzier** [comment]
All these definitions that only call `ResizeFrame` or `ResizeFrameWithPage` feel like they should be handled by a base class instead. Can't this be moved to where the `OnResize` function is called?

**Comment by Gymnasiast** [comment]
With the current state of the window system, I really doubt that. The windows still have too many differences and handle too much of their own stuff for that.

**Comment by MarkOtzen** [comment]
Some user feedback; I wasn't a fan of this defaults-change on macOS since it went against what I'm used to playing this game, but was glad to see it at least was revertible.

**Pull request merged**